src/util/MetadataController.ts

```diff
@@ -69,7 +70,7 @@ export class MetadataController {
    * @returns The Readable object.
    */
   public generateReadableFromQuads(quads: Quad[]): Readable {
```

Comment:
I think this function name (and corresponding comment) is misinformed: isn't it `serializeQuads`? Because that is what it seems to be doing.

Reply:
suggestion for the comment: "Helper function for serializing an array of quads, with as result a Readable object." Or something else?